package.json

```
@@ -21,6 +22,8 @@
   },
   "homepage": "https://github.com/RaisinTen/aes-crypto-js#readme",
   "devDependencies": {
-    "crypto-js": "^4.2.0"
+    "crypto-js": "^4.2.0",
+    "@types/node": "^24.0.3",
+    "typescript": "^5.8.3"
```

Using tsd would be convenient to test the argument types as well as the return type of the function in the .d.ts file
```suggestion
    "tsd": "^0.32.0"
```